build.gradle

```diff
@@ -405,15 +414,14 @@ subprojects { subproj ->
 			endWithNewline()
 		}
 
-		format 'groovy', {
+		groovy {
 			target 'src/**/*.groovy'
-			indentWithTabs()
+			licenseHeaderFile headerText, "package "
+			importOrderFile importOrderConfig
+			greclipse().configFile formatterJavaConfig, formatterGroovyConfig
+
 			trimTrailingWhitespace()
 			endWithNewline()
-			licenseHeaderFile headerFile, "package "
-
-			replaceRegex 'class-level Javadoc indentation fix', /^\*/, ' *'
-			replaceRegex 'nested Javadoc indentation fix', /\t\*/, '\t *'
```

Comment:
The GrEclipse formatter is based on a quite old JDT formatter and definitely not bug-free. But these simple things are working without any problems.